R/get_calib_periods.R

```diff
@@ -0,0 +1,41 @@
+#'@title Splits time period into calibration and validation period
+#'
+#'@description Reads in nml-file and splits the time period in a calibration and a validation time period to avoid over-fitting
+#'@param nml String of the GLM namelist file, in most cases this is 'glm3.nml'
+#'@param ratio Double of the ratio between calibration and validation, default is 1 which refers to calibration:validation 1:1, 2 would
+#' mean that calibration:validation would be 2:1
+#'@keywords methods
+#'@seealso \code{\link{get_calib_setup}, \link{calibrate_sim}, \link{get_calib_init_validation}}
+#'@author
+#'Robert Ladwig
+#'
+#'@examples
+#'calib_periods <- get_calib_periods(system.file("extdata/", "glm3.nml", package = "glmtools"), ratio = 1)
+#'@export
+get_calib_periods <- function(nml, ratio = 1){
```

Comment:
best to be consistent with the naming. here it is `nml`, but above it is `nml_file`. I think both are the same thing

Reply:
Good point. Changing these. Also, some discrepancy between nml.file and nml_file (going with the latter).

Reply:
agreed. `nml_file` seems preferred